docs/intro.rst

```diff
@@ -47,10 +48,40 @@ coroutine returns.
 Here's a corresponding client example.
 
 .. literalinclude:: ../example/client.py
+    :emphasize-lines: 7-8
 
 Using :func:`connect` as an asynchronous context manager ensures the
 connection is closed before exiting the ``hello`` coroutine.
 
+Secure example
+--------------
+
+Secure WebSocket connections improve confidentiality and also reliability
+because they reduce the risk of interference by bad proxies.
+
+The WSS protocol is to WS what HTTPS is to HTTP: the connection is encrypted
+with TLS. WSS requires TLS certificates like HTTPS.
+
+Here's how to adapt the server example to provide secure connections, using
+APIs available in Python ≥ 3.6.
+
+Refer to the documentation of the :mod:`ssl` module for configuring the
+context securely or adapting the code to older Python versions.
+
+.. literalinclude:: ../example/secure_server.py
+    :emphasize-lines: 20-21
+
+Here's how to adapt the client, also on Python ≥ 3.6.
+
+.. literalinclude:: ../example/secure_client.py
+    :emphasize-lines: 13-14
+
+This example builds a context because it uses a self-signed certificate.
+
+To connect to a secure WebSocket server with a valid certificate (i.e. signed
+by a CA that your Python installation trusts), simply pass ``ssl=True`` to
+``connect`` instead of building a context.
+
```

Comment:
My understanding is that for adapting the example to the case where a valid certificate is used, we would still need an `ssl_context` for the server side, and only the client side needs to be altered, right ?

If that's not what you mean to say, then the wording is confusing; if it is what you mean, I suggest that you make this a little more explicit, like with

>(i.e. *blabla*), alter the client code and simply pass ``ssl=True`` to ``connect`` instead of building a context.

Reply:
You're correct. I'll clarify.